CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ Release notes are available on [github][notes].
 [pub-semver-readme]: https://pub.dartlang.org/packages/pub_semver
 [notes]: https://github.com/java-james/flutter_dotenv/releases
 
+#### 3.2.0-nullsafety.0
```

Comment:
Change to 4.0.0, mention breaking null-safety changes

Reply:
Updated!